src/commonMain/kotlin/exchange.dydx.abacus/state/manager/utils/Payloads.kt

```diff
@@ -130,3 +130,13 @@ data class HumanReadableTransferPayload(
     val amount: String,
     val recipient: String,
 )
+
+@JsExport
+@Serializable
+data class HumanReadableChainPayload(
```

Comment:
`HumanReadablePayload` typically refers to the payload to be submitted with on-chain calls (i.e., calls to v4-clients functions).  Since this is just a normal struct,  can we remove rename this to something like `TransferChainInfo`?

Reply:
sounds good tom e!